pkg/pointer/pointer.go

```diff
@@ -0,0 +1,24 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2022 Red Hat, Inc.
+ *
+ */
+
+package pointer
```

Comment:
Nice! Why not under pkg/util? Even in k8s it is under utils - `k8s.io/utils/pointer`.

Reply:
I think any generic name, like `utils`, ends up as a default target for helpers of unrelated topics.
Then, when one needs to use one small thing (like this one), it will fetch in (as dependencies) all the others.

I prefer not to contribute to this garbage container.

I also do not consider k8s as a good example here, the `utils` there is even worse, it is a repository.